cirras-underwriting-war/cirras-underwriting-war/src/main/angular/package.json

```diff
@@ -65,7 +65,7 @@
     "ngx-trim": "^1.0.1",
     "puppeteer": "^1.14.0",
     "rxjs": "^6.3.3",
-    "sass": "^1.62.0",
+    "sass": "^1.78.0",
```

Comment:
Sass is an extension to CSS. I ran npm -update after changing the .npmrc file, hoping that this will resolve the issues of connecting to the Vivid artifactory instead of the gov't one on my local machine. The command also updated the local packages in the project, where sass was the only one which had a new version available. Regular updates of the local packages is recommended.